plsync/sites.py

```diff
@@ -114,6 +114,7 @@
     makesite('lga04','173.205.4.64',   '2001:0668:001F:0022::','New York_NY', 'US', 40.766700, -73.866700, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
     makesite('lga05','4.35.94.0',      '2001:1900:2100:14::',  'New York_NY', 'US', 40.766700, -73.866700, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
     makesite('lga06','128.177.119.192','2001:438:fffd:2b::',   'New York_NY', 'US', 40.766700, -73.866700, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
+    makesite('lga07','66.151.223.128','2600:c0f:2002::',   'New York_NY', 'US', 40.766700, -73.866700, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
```

Comment:
Are you sure that this is a 4 node site, and that it is 64 bit? You mention in the PR comment that this is not new hardware, but just a moving the old hardware.  If you take a look at the [commit where lga01 got removed](https://github.com/m-lab/operator/commit/dec505c1077621cea3751216294b25bce19b19c5), it doesn't have count=4, nor arch='x86_64'.  Probably this new site should be the same?